src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
+function calculateMenAverageAge(people, century = 0) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const man = (person) => (person.sex === 'm');
```

Comment:
this is not a man, this is a function, name the variables that are more suitable for their purpose (everywhere)